Source/WebCore/platform/graphics/FontCascadeCache.h

```diff
@@ -96,12 +96,12 @@ struct FontDescriptionKeyRareData : public RefCounted<FontDescriptionKeyRareData
     FontVariationSettings m_variationSettings;
     FontVariantAlternates m_variantAlternates;
     FontPalette m_fontPalette;
-    std::optional<float> m_fontSizeAdjust;
+    std::optional<FontSizeAdjust> m_fontSizeAdjust;
 };
 
 inline void add(Hasher& hasher, const FontDescriptionKeyRareData& key)
 {
-    add(hasher, key.featureSettings(), key.variationSettings(), key.variantAlternates(), key.fontPalette(), key.fontSizeAdjust());
+    add(hasher, key.featureSettings(), key.variationSettings(), key.variantAlternates(), key.fontPalette(), (*key.fontSizeAdjust()).value);
```

Comment:
Does the hasher need to take in account the metric too?

Reply:
How does the hasher know the fontSizeAdjust isn't nullopt?